src/components/organisms/AppRouter/AdminV3Subrouter.tsx

```diff
@@ -27,6 +27,8 @@ import { AdminVenueView } from "components/organisms/AdminVenueView";
 export const AdminV3Subrouter: React.FC = () => (
   <Switch>
     <Route
+      exact
+      strict
```

Comment:
[NOTE]

This is resolved with re-arranging of routes in a next PR. Good enough for now.